src/Timing.cc

```diff
@@ -385,4 +385,44 @@ float Timing::dynamicPower(odb::dbInst* inst, sta::Corner* corner)
   return (power.internal() + power.switching());
 }
 
+void Timing::makeEquivCells()
```

Comment:
Rather than copying this code from Resizer I think it would be better to just make it a public method there and call it here.

Reply:
I think makeEquivCells and equivCells should be placed together, either in Timing.cc or in Resizer.cc, However, since equivCells is from sta, so I put both of them in Timing.cc.